sdk/cosmos/azure-cosmos/azure/cosmos/aio/database.py

```diff
@@ -532,26 +584,34 @@ async def create_user(self, body, **kwargs):
             client_connection=self.client_connection, id=user["id"], database_link=self.database_link, properties=user
         )
 
-    def get_user_client(self, user_id):
-        # type: (str) -> UserProxy
+    def get_user_client(self, user: Union[str, UserProxy, Dict[str, Any]]) -> UserProxy:
```

Comment:
I see this param was always documented as `user`, although it was previously `user_id` - what did the sync client have?

Reply:
user as well - this was a change we reverted since it didn't really make sense to edit to begin with, even if we only end up using the id